src/components/application_manager/test/commands/mobile/subscribe_button_request_test.cc

```diff
@@ -116,10 +122,13 @@ TEST_F(SubscribeButtonRequestTest, Run_SubscriptionNotAllowed_UNSUCCESS) {
 }
 
 TEST_F(SubscribeButtonRequestTest, Run_UiIsNotSupported_UNSUCCESS) {
-  CommandPtr command(CreateCommand<SubscribeButtonRequest>());
+  MessageSharedPtr msg = CreateMessage();
+  (*msg)[am::strings::params][am::strings::connection_key] = kConnectionKey;
+  CommandPtr command(CreateCommand<SubscribeButtonRequest>(msg));
 
-  MockAppPtr app(CreateMockApp());
-  ON_CALL(mock_app_manager_, application(_)).WillByDefault(Return(app));
+  MockAppPtr mock_app(CreateMockApp());
+  ON_CALL(mock_app_manager_, application(kConnectionKey))
```

Comment:
@wolfylambova22 You use it almost in all test cases. May be it's worth to move it to SetUp or ctor? Or even create InitGetters() method and put all ON_CALLs there.